gitpod-io/website
# [pricing] Same Discounts on teams pricing

## Description

Same Discounts apply to Teams Pricing (this PR depends on the deployment of [PR#7800](https://github.com/gitpod-io/ops/pull/7800), _ETA: Tomorrow_)

## Related Issue(s)



Fixes #3289 

## How to test



Open [`/pricing`](https://deploy-preview-3297--gitpod-kumquat.netlify.app/pricing), Test Pricing Calculator & See Feature Table (teams section)
